packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -88,30 +73,35 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/db
 %{gem_libdir}
 %{gem_instdir}/locale
+%exclude %{gem_instdir}/package.json
+%exclude %{gem_instdir}/webpack
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
 %{foreman_assets_plugin}
 %{foreman_assets_foreman}
+%{foreman_webpack_plugin}
+%{foreman_webpack_foreman}
 
 %files doc
 %doc %{gem_docdir}
 %doc %{gem_instdir}/README.md
 %{gem_instdir}/Rakefile
 %{gem_instdir}/test
 
-%post
-/sbin/semanage port -a -t http_port_t -p tcp 8006 &> /dev/null || :
-
-%postun
-if [ $1 -eq 0 ] ; then
-  /sbin/semanage port -d -t http_port_t -p tcp 8006 &> /dev/null || :
-fi
-
```

Comment:
I think this should be kept

Reply:
True, changed it, thanks for paying attention!